website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing a user from hovering, focusing, or clicking them. For users with assistive technology (e.g., a screen reader), disabled elements are almost entirely ignored, including the text within.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users to understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+### Upgrading
+
+When premium features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity.
+
+An example of this is Figma's "create branch" function that is only available at the enterprise tier. Clicking "create a branch" in Figma opens a marketing modal that entices users to upgrade to gain access to this feature.
+
+![Screen shot of figma's modal showing information about how to upgrade to enterprise to gain access to creating branches.](/assets/patterns/disabled-patterns/upgrade-modal-example.png)
+
+
+### Service outage
+
+In instances where outages may occur, provide a clear message to users on what they can do and add a link to [our status page](https://status.hashicorp.com/).
+
+An example of this is Instagram's login page. When a user has no connection to their servers, a message will appear on submit.
+
+![Displaying Instagram's login page with an error below notifying the user they have no connection to instagram.](/assets/patterns/disabled-patterns/service-outage-example.png)
+
+### Incomplete flow
+
+In a form, regardless of whether all of the fields have been filled out, always enable the submit button. When we disable the only way to move forward, users are often left confused and frustrated. Instead, allow users to understand what erorrs took place so that they can fix them and proceed in the form's completion.
+
+In this example, creating a cluster has required inputs. Clicking "create cluster" will display errors where applicable, allowing the user to understand where the problems are to remediate them.
+
+![Image showing an input field with an error explaining what was missed.](/assets/patterns/disabled-patterns/incomplete-flow-example.png)
+
+### Quota limitation
+
+When a quota limit has been hit, hide the action and replace it with guidance and a call to action to increase the quota (if possible). If the user does not have permissions to upgrade, direct them to an admin instead.
```

Comment:
**suggestion (non-blocking):** More verbose, but provides more clarity. 
```suggestion
When a quota has been reached, hide the action allowing them to create more objects. If a user can increase the quota, replace the original action with an action that allows them to do so. If the user cannot increase the quota, e.g., they don't have permission, direct them to an admin.
```

**question:** Should we really direct them to an admin or advocate for hiding the action altogether?

Reply:
That's a good point, ill update to say "direct them to a form" instead of an admin.

Reply:
Updated wording.